### PR TITLE
Update website pricing page.

### PR DESCRIPTION
**Pull request by EvanHStanton**
# Notes for blog or docs authors:

-   The `image: ` field in frontmatter of blogs should always be a `png` (`svg` will not work for social previews)
-   You must also duplicate the asset used in `image :` inside of the [public folder](https://github.com/tembo-io/website/tree/main/public)
-   Please write a `description: ` in the frontmatter of any new blog or doc
-   Read more [here](https://github.com/tembo-io/website?tab=readme-ov-file#writing-a-blog-post-%EF%B8%8F)

### CODE REVIEW

**Review by sarahedkins — APPROVED** [review]

**Review by DarrenBaldwin07 — APPROVED** [review]

File: src/components/PricingCards.astro [hunk 1]
```diff
@@ -43,7 +45,7 @@ const selfhostedFeatures = [
 	<h2
 		class='font-semibold text-pricingLightBlue text-[28px] leading-[35px] tracking-[0.54px]'
 	>
-		Hobby
+		Cloud: Hobby
```
[the remark this answers is not in the document]
Comment (samay-sharma) [reply src/components/PricingCards.astro:48]: Let's have the tagline for this be: For personal projects, exploration and non-critical applications

File: src/components/PricingCards.astro [hunk 2]
```diff
@@ -92,7 +94,7 @@ const selfhostedFeatures = [
 		<h2
 			class='font-semibold text-pricingGreen text-[28px] leading-[35px] tracking-[0.54px]'
 		>
-			Production
+			Cloud: Production
```
[the remark this answers is not in the document]
Comment (samay-sharma) [reply src/components/PricingCards.astro:97]: Tagline: For production applications requiring performance and SLA

File: src/components/PricingCards.astro [hunk 3]
```diff
@@ -8,32 +8,34 @@ import ProductionPricing from '../components/ProductionPricing';
...
+	'On-Demand Instance Deployment and Resizing',
+	'Up to 16 vCPU & 64 GB Memory',
+	'Up to 2 TB Storage',
 	'99.95% SLA',
-	'Apps',
+	'Tembo Team Consultation Support',
 ];
 
 const selfhostedFeatures = [
-	'Private Data Plane',
```
Comment (samay-sharma) [inline src/components/PricingCards.astro:31]: Let's have this list be:
* Full power of Tembo
* Enterprise-grade Kubernetes operator
* Run on-prem or on any cloud provider
* Advanced Enterprise Security
* Private Networking
* Custom infrastructure and security configuration
* Dedicated Onboarding Engineer
* 24x7x365 Technical Support

File: src/pages/pricing.astro [hunk 4]
```diff
@@ -39,12 +39,12 @@ const platformIncluded = [
 const generalRows = [
 	['App support', true, true, true],
 	['200+ extensions', true, true, true],
-	['Point in time recovery', true, true, true],
 	['Monitoring & Alerting', true, true, true],
 	['High Availability', false, true, true],
-	['Scale up storage', false, true, true],
-	['Instance Re-sizing', false, true, true],
-	['Uptime SLA', 'N/A', '99.95% with HA', '99.95% with HA'],
+	['On-Demand Instance Deployment', false, true, true],
```
Comment (samay-sharma) [inline src/pages/pricing.astro:44]: This is hard to explain and understand. Let's remove.

File: src/pages/pricing.astro [hunk 5]
```diff
@@ -39,12 +39,12 @@ const platformIncluded = [
...
+	['Adjust Scale & Compute', false, true, true],
```
Comment (samay-sharma) [inline src/pages/pricing.astro:45]: Scale storage and compute

File: src/pages/pricing.astro [hunk 6]
```diff
@@ -39,12 +39,12 @@ const platformIncluded = [
 const generalRows = [
 	['App support', true, true, true],
 	['200+ extensions', true, true, true],
```
Comment (samay-sharma) [inline src/pages/pricing.astro:41]: Add Stacks

File: src/pages/pricing.astro [hunk 7]
```diff
@@ -39,12 +39,12 @@ const platformIncluded = [
...
-	['Point in time recovery', true, true, true],
 	['Monitoring & Alerting', true, true, true],
 	['High Availability', false, true, true],
-	['Scale up storage', false, true, true],
-	['Instance Re-sizing', false, true, true],
-	['Uptime SLA', 'N/A', '99.95% with HA', '99.95% with HA'],
+	['On-Demand Instance Deployment', false, true, true],
+	['Adjust Scale & Compute', false, true, true],
+	['Uptime SLA', 'N/A', '99.95% with HA', 'Custom'],
+	['Fully-Managed', true, true, false],
```
Comment (samay-sharma) [inline src/pages/pricing.astro:47]: Add two points:
Cloud provider support: AWS, AWS, All
Customizable Security and Compliance : false false true

File: src/components/PricingCards.astro [hunk 8]
```diff
@@ -134,7 +136,7 @@ const selfhostedFeatures = [
 	</h2>
 	<span
 		class='h-[38px] block mt-[15px] font-secondary font-light text-white text-base leading-[19px] opacity-80'
```
[the remark this answers is not in the document]
Comment (DarrenBaldwin07) [reply src/components/PricingCards.astro:138]: Screenshot 2024-06-20 at 9 46 26 AM

File: src/pages/pricing.astro [hunk 9]
```diff
@@ -32,33 +32,35 @@ const generalIncluded = [
...
-	['Uptime SLA', 'N/A', '99.95% with HA', '99.95% with HA'],
+	['Scale Storage & Compute', false, true, true],
+	['Uptime SLA', 'N/A', '99.95% with HA', 'Custom'],
+	['Fully-Managed', true, true, false],
+	['Cloud Provider Support', 'AWS', 'AWS', 'All'],
+	['Customizable Security and Compliance', false, false, true],
 ];
 
 const platformRows = [
 	['Hobby Tier Instances', '3', 'Unlimited', 'Unlimited'],
```
Comment (samay-sharma) [inline src/pages/pricing.astro:53]: Change this to number of instances

File: src/components/Faqs.tsx [hunk 10]
```diff
@@ -4,7 +4,8 @@ import Accordion from './Accordion';
 const faqs = [
 	{
 		heading: 'When will I be billed?',
-		content: 'Our Production plan charges based on usage. You will receive your involve at the end of each month.',
+		content:
+			'Plans on Tembo Cloud are charged based on usage; you will receive your invoice at the end of each month. Self Hosted deployments can vary, which is why we encourage you to contact us so we can work together to find the most appropriate plan.',
```
Comment (samay-sharma) [inline src/components/Faqs.tsx:8]: Include link to contact us form where we mention Contact Us.

File: src/components/Faqs.tsx [hunk 11]
```diff
@@ -14,29 +15,64 @@ const faqs = [
 	},
 	{
 		heading: 'Are annual plans available?',
-		content:
-			'Yes, we do provide annual plans which offer a significant discount over the list price. Please reach out to brad@tembo.io to learn more.',
```
Comment (samay-sharma) [inline src/components/Faqs.tsx:18]: Let's redirect to contact us form instead of email.

File: src/components/Faqs.tsx [hunk 12]
```diff
@@ -14,29 +15,64 @@ const faqs = [
...
+				We've designed our{' '}
+				<a
+					href={
+						'https://tembo.io/docs/product/software/tembo-self-hosted/overview/'
+					}
+					className='underline'
+				>
+					Self Hosted
+				</a>{' '}
+				offering to be both modular and flexible. So long as you've got
```
Comment (samay-sharma) [inline src/components/Faqs.tsx:60]: Correct to: "I have a preferred cloud provider or region that I don't see on Tembo Cloud. What can I do?"

We are continuously adding new regions to our Tembo Cloud offering. Please go to [our roadmap](https://roadmap.tembo.io/roadmap) to request the regions or cloud providers you'd like. You can also use our Self-Hosted offering which allows you to run Tembo on any cloud provider or region which supports deploying a Kubernetes cluster.

**Review by samay-sharma — APPROVED** [review]
Please make these final edits and then merge.

**Review by DarrenBaldwin07 — APPROVED** [review]

**Pull request merged**